wallet/lib/services/repository/repository.dart

```diff
@@ -1859,7 +1859,7 @@ class RepositoryImp implements Repository {
     } on Failure catch (_) {
       return Left(_);
     } on Exception catch (_) {
-      recordErrorInCrashlytics(_);
+      // recordErrorInCrashlytics(_);
```

Comment:
Should there be dead code here?

Reply:
Yes i silenced error. Until chain starts receiving custom events.